neo_batterylevelshutdown/displays.py

```diff
@@ -1,159 +1 @@
                if self._curPage == self.pages[-1]:
                    self._curPage = self.pages[0]
                else:
                    self._curPage = \
                        self.pages[self.pages.index(self._curPage) + 1]

            # draw the page while holding the lock, so that it doesn't change
            #  underneath us
            self._curPage.draw_page()
            logging.debug("Transitioned to page %s", self._curPage)

    def moveBackward(self):
        with self._curPageLock:
            logging.debug("Current page is %s", self._curPage)
            if self._curPage not in self.pages:
                # Always start with the starting page if the screen went off
                #  or if we were showing the low battery page
                self._curPage = self.pages[self.STARTING_PAGE_INDEX]
            else:
                # move backwards in the page list
                # If we're at the start of the page list, go to the start
                if self._curPage == self.pages[0]:
                    self._curPage = self.pages[-1]
                else:
                    self._curPage = \
                        self.pages[self.pages.index(self._curPage) - 1]

            # draw the page while holding the lock, so that it doesn't change
            #  underneath us
            self._curPage.draw_page()
            logging.debug("Transitioned to page %s", self._curPage)

    def showLowBatteryWarning(self):
        if self._curPage == self.low_battery_page:
            # nothing to do
            return

        with self._curPageLock:
            logging.debug("Current page is %s", self._curPage)
            self._curPage = self.low_battery_page
            self._curPage.draw_page()
            logging.debug("Transitioned to page %s", self._curPage)

    def hideLowBatteryWarning(self):
        if self._curPage == self.low_battery_page:
            self.powerOffDisplay()

    def powerOffDisplay(self):
        if self._curPage == self.blank_page:
            # nothing to do
            return

        with self._curPageLock:
            logging.debug("Current page is %s", self._curPage)
            self._curPage = self.blank_page
            self._curPage.draw_page()
            logging.debug("Transitioned to page %s", self._curPage)

    # Ideally this should be a page, like the low battery page
    def drawLogo(self):
        dir_path = os.path.dirname(os.path.abspath(__file__))
        img_path = dir_path + '/assets/connectbox_logo.png'
        logo = Image.open(img_path).convert("RGBA")
        fff = Image.new(logo.mode, logo.size, (255,) * 4)
        background = Image.new("RGBA", self.display_device.size, "black")
        posn = ((self.display_device.width - logo.width) // 2, 0)
        img = Image.composite(logo, fff, logo)
        background.paste(img, posn)
        self.display_device.display(
            background.convert(self.display_device.mode)
        )

```

Comment:
Another line ending issue

Reply:
I see the problem, but I have no clue how best to address it.  I believe I have fixed the issue for future purposes, by changing git settings.  But for this one, I am honestly at a loss.  

I would be all ears if anyone has any suggestions, if not, I'll be hitting google up in the morning. 

I'll update the notes once I get a good fix on this.

Reply:
sounds like it is probably an issue with your editor. Which one are you using?

Reply:
I believe the code has been straightened out and ready to be tested.